Tools/SonarQube/azure/build.yaml

```diff
@@ -0,0 +1,203 @@
+pool:
+  name: 'HealthGateway'
+  
```

Comment:
We need to have this scheduled as per https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml